### PR TITLE
Update getting started to use Fargate

### PR DESCRIPTION
**Pull request by chrisnegus**
**1. Issue, if available:**
[Issue #1812](https://github.com/aws/karpenter/issues/1812), [Issue #1763](https://github.com/aws/karpenter/issues/1763), [Issue #1365](https://github.com/aws/karpenter/issues/1365),

**2. Description of changes:**
Update the Karpenter [Getting started with eksctl guide](https://karpenter.sh/preview/getting-started/getting-started-with-eksctl/) with an example that creates a Fargate profile in which to run Karpenter.

I did it the most simple way I could think of. So let me know if I should have gone about it a different way.

**3. How was this change tested?**
I tested the whole eksctl getting started guide using both cluster creation examples.

**4. Does this change impact docs?**
- [ ] Yes, PR includes docs updates
- [X] Yes, issue opened: *see issues above*
- [ ] No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: website/content/en/preview/getting-started/getting-started-with-eksctl/scripts/step02-create-cluster.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@ kind: ClusterConfig
 metadata:
   name: ${CLUSTER_NAME}
   region: ${AWS_DEFAULT_REGION}
-  version: "1.21"
+  version: "1.22"
```
Comment (FernandoMiguel) [inline website/content/en/preview/getting-started/getting-started-with-eksctl/scripts/step02-create-cluster.sh:8]: this is a good change, but it is out of scope of this PR

Comment (chrisnegus) [reply website/content/en/preview/getting-started/getting-started-with-eksctl/scripts/step02-create-cluster.sh:8]: True. I'll add it to a separate PR.

**Review by FernandoMiguel — DISMISSED** [review]

File: website/content/en/preview/getting-started/getting-started-with-eksctl/scripts/step02-create-cluster-fargate.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+---
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+metadata:
+  name: ${CLUSTER_NAME}
+  region: ${AWS_DEFAULT_REGION}
+  version: "1.21"
+  tags:
+    karpenter.sh/discovery: ${CLUSTER_NAME}
+managedNodeGroups:
```
Comment (suket22) [inline website/content/en/preview/getting-started/getting-started-with-eksctl/scripts/step02-create-cluster-fargate.sh:11]: Is this Managed Nodegroup necessary when using Fargate?

Comment (chrisnegus) [reply website/content/en/preview/getting-started/getting-started-with-eksctl/scripts/step02-create-cluster-fargate.sh:11]: Yeah, I should have removed that. I am having trouble getting just Fargate to work at the moment. I'm debugging right now, so I don't want to merge this yet.

File: website/content/en/preview/getting-started/getting-started-with-eksctl/_index.md [hunk 3]
```diff
@@ -55,13 +55,19 @@ Also set the following environment variables to store commonly used values.
...
+The first uses [AWS EKS managed node groups](https://docs.aws.amazon.com/eks/latest/userguide/managed-node-groups.html) for the control plane and Karpenter, while the second splits off the Karpenter controller to use a Fargate profile.
+
+**Example 1: Create basic cluster**
 
 {{% script file="./content/en/{VERSION}/getting-started/getting-started-with-eksctl/scripts/step02-create-cluster.sh" language="bash"%}}
 
-This guide uses [AWS EKS managed node groups](https://docs.aws.amazon.com/eks/latest/userguide/managed-node-groups.html) to host Karpenter.
+**Example 2: Create basic cluster with Karpenter on Fargate**
+
+{{% script file="./content/en/{VERSION}/getting-started/getting-started-with-eksctl/scripts/step02-create-cluster-fargate.sh" language="bash"%}}
```
[the remark this answers is not in the document]
Comment (chrisnegus) [reply website/content/en/preview/getting-started/getting-started-with-eksctl/_index.md:69]: After making the fix you suggested I now see those coredns messages. Should we still add a pointer to the fargate docs on updating CoreDNS or is that something that could perhaps go into the Troubleshooting guide?

File: website/content/en/preview/getting-started/getting-started-with-eksctl/scripts/step02-create-cluster-fargate.sh (hunk 2 above)
Comment (chrisnegus) [reply website/content/en/preview/getting-started/getting-started-with-eksctl/scripts/step02-create-cluster-fargate.sh]: I removed the manageNodeGroups from the karpenter example.

**Review by tzneal — APPROVED** [review]
LGTM, tested this locally twice.

File: website/content/en/preview/getting-started/getting-started-with-eksctl/scripts/step02-create-cluster-fargate.sh [hunk 4]
```diff
@@ -0,0 +1,20 @@
+eksctl create cluster -f - << EOF
```
Comment (FernandoMiguel) [inline website/content/en/preview/getting-started/getting-started-with-eksctl/scripts/step02-create-cluster-fargate.sh:1]: mu concern with this, is that this is now handled by cloudformation and nobody is going to destroy this, leaving garbage in the user account

**Pull request merged**